### PR TITLE
Move docs from managed tenants GitHub repo

### PR DESCRIPTION
**Pull request by garrettthomaskth**

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ weight: 10
     <p>
       Guides and resources to help you get a service into production and the hands of customers.
```
Comment (Nanyte25) [inline content/en/_index.md]: @garrettthomaskth this text should be ` Guides and resources to help you get a service into production and into the hands of customers.`

File: content/en/docs/internal-documentation/ic.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Surfaces
+
+* Slack channels:
+  * `#sd-mt-sre-info`
+  * `#mt-cs-sre-teamchat`
+  * `#mt-cs-sre-teamhandover`
+  * `#forum-managed-tenants`
+* Mailing lists:
+  * managed-tenants@redhat.com
```
Comment (nikhil-thomas) [inline content/en/docs/internal-documentation/ic.md]: shall we make it `mt-cs-sre@redhat.com` 

ref: [slack thread](https://redhat-internal.slack.com/archives/C045X0P531D/p1675099835235859)

Comment (Nanyte25) [reply content/en/docs/internal-documentation/ic.md]: yes I would @garrettthomaskth

File: content/en/docs/addons-flow/architecture.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+## Deprecated SelectorSyncSet Installation
+
+![Data Flows](/architecture_data_flow.png)
+When you click "Install" in the OCM Web UI, under the hood, OCM will apply
+a label to the corresponding `ClusterDeployment` object in Hive.
+
+That label is the same label used in the `SelectorSyncSet` as a `matchLabel`.
+
+With the `ClusterDeployment` label now matching the `SelectorSyncSet` label,
+Hive will apply all the objects in the `SelectorSyncSet` to the target cluster:
```
Comment (Nanyte25) [inline content/en/docs/addons-flow/architecture.md:76]: Can we use the wording for HIve here? @garrettthomaskth I suppose we can as it has a public repo

File: content/en/docs/creating-addons/monitoring/slo_dashboards.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
+---
+title: "SLO Dashboards"
+date: 2022-12-15T00:53:51+01:00
+---
+
+Development teams are requested to co-maintain, with the MT-SRE Team, SLO
```
Comment (Nanyte25) [inline content/en/docs/creating-addons/monitoring/slo_dashboards.md]: @garrettthomaskth `Development teams are required to co-maintain, in conjunction with the MT_SRE Team, SLO`

File: content/en/docs/creating-addons/top-level-operator/customer-notifications.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Customer Notifications"
+date: 2022-12-15T00:53:51+01:00
```
Comment (Nanyte25) [inline content/en/docs/creating-addons/top-level-operator/customer-notifications.md:3]: Should we mention `service logs` and `Pendo` or `Statuspage` here?

File: content/en/_index.md [hunk 6]
```diff
@@ -8,44 +8,37 @@ weight: 10
 
 <div class="cover-section">
   <h1 class="display-1">Red Hat Layered Products!</h1>
-  <p class="lead">Managing managed services on managed OpenShift!</p>
+ <p class="lead">Managing managed services on managed OpenShift!</p>
```
Comment (Nanyte25) [inline content/en/_index.md]: @garrettthomaskth `Providing managed services on managed Openshift`?

Comment (garrettthomaskth) [reply content/en/_index.md]: Hahah :sweat_smile: yes much better

File: content/en/docs/addons-flow/architecture.md (hunk 3 above)
Comment (garrettthomaskth) [reply content/en/docs/addons-flow/architecture.md:76]: Hey @Nanyte25 ! Sorry what exactly do you mean by the wording for Hive?

File: content/en/docs/internal-documentation/ic.md (hunk 2 above)
Comment (garrettthomaskth) [reply content/en/docs/internal-documentation/ic.md]: Yes, great catch, thank you!

File: content/en/_index.md (hunk 1 above)
Comment (garrettthomaskth) [reply content/en/_index.md]: :+1:

File: content/en/docs/creating-addons/top-level-operator/customer-notifications.md (hunk 5 above)
Comment (garrettthomaskth) [reply content/en/docs/creating-addons/top-level-operator/customer-notifications.md:3]: Yes definitely! But I actually don't even know about `service logs` or `Pendo`! Could you share some links if you have them please? :)

File: content/en/docs/addons-flow/architecture.md (hunk 3 above)
Comment (Nanyte25) [reply content/en/docs/addons-flow/architecture.md:76]: should we be mentioning the word `Hive` specifically our `internal shards` its fine don't worry about it I was over thinking things? @garrettthomaskth

**Review by Nanyte25 — APPROVED** [review]
@garrettthomaskth I am happy to approve once the small changes are done.

**Review by Nanyte25 — APPROVED** [review]

**Pull request merged**